File: 1/tarifa/tarifa.go
// https://open.kattis.com/problems/tarifa

package main

import "fmt"

func main() {

    // declare some strongly typed vars
    var monthly, cases, extra, usage int

    extra = 0

    // input reads in like C
    fmt.Scanln(&monthly)
    fmt.Scanln(&cases)

    for i := 0; i < cases; i++ {

        fmt.Scanln(&usage)

        extra += monthly - usage

    }

    extra += monthly

    fmt.Println(extra)
}

File: 1/tarifa/tarifa.java
// https://open.kattis.com/problems/tarifa

import java.util.Scanner;

public class tarifa {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int cap = scan.nextInt();
        int n = scan.nextInt();
        int extra = 0;
        for(int i=0; i<n; i++){
            int x = scan.nextInt();
            extra = extra + cap - x;
        }
        System.out.println(extra+cap);
    }

}
